src/utils/normalizer.js

```diff
@@ -40,10 +45,10 @@ export function normalWidthMargin() {
 
 /**
  * Function to fix the height of Status Bar ONLY in the iPhoneX
- * @param size
+ * @param {number} size The actual device's screen size
  * @returns {number} The correct size for the Status Bar
  */
-export function isiPhoneXStatusBar(size) {
+export function isiPhoneXStatusBar(size: number) {
   if (Platform.OS === 'ios' && (deviceHeight === 812 || deviceWidth === 812)) {
```

Comment:
I suggest to extract this check into separate function called `isiPhoneX` for example, because it's already used twice.

Reply:
Done